Functions.Templates/Resources/Resources.resx

```diff
@@ -831,6 +831,9 @@
   <data name="BlobTrigger_description" xml:space="preserve">
     <value>A function that will be run whenever a blob is added to a specified container</value>
   </data>
+  <data name="OrchestrationTrigger_description" xml:space="preserve">
+    <value>A function that will be run whenever an orchestration is added to a specified container</value>

```

Comment:
haha, I think the container part is only for storage-specific triggers :)
How about this description?
```suggestion
    <value>A stateful serverless workflow defined in code.</value>

```